doc/amazon_sagemaker_processing.rst

```diff
@@ -0,0 +1,126 @@
+.. sectnum::
+
+##############################
+Amazon SageMaker Processing
+##############################
+
+
+Amazon SageMaker Processing allows you to run steps for data pre- or post-processing, feature engineering, data validation, or model evaluation workloads on Amazon SageMaker.
+
+.. contents::
+
+Background
+==========
+
+Amazon SageMaker lets developers and data scientists train and deploy machine learning models. With Amazon SageMaker Processing, you can run processing jobs on for data processing steps in your machine learning pipeline, which accept data from Amazon S3 as input, and put data into Amazon S3 as output.
+
+.. image:: ./amazon_sagemaker_processing_image1.png
+
+Setup
+=====
+
+The fastest way to run get started with Amazon SageMaker Processing is by running a Jupyter notebook. You can follow the `Getting Started with Amazon SageMaker`_ guide to start running notebooks on Amazon SageMaker.
+
+.. _Getting Started with Amazon SageMaker: https://docs.aws.amazon.com/sagemaker/latest/dg/gs.html
+
+You can run notebooks on Amazon SageMaker that demonstrate end-to-end examples of using processing jobs to perform data pre-processing, feature engineering and model evaluation steps. See `Learn More`_ at the bottom of this page for more in-depth information.
+
+
+Data Pre-Processing and Model Evaluation with Scikit-Learn
```

Comment:
scikit-learn's docs leave "scikit-learn" lowercase as far as I can tell while Wikipedia capitalizes it as "Scikit-learn"